CHANGELOG.rst

```diff
@@ -5,6 +5,18 @@ Changelog
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 
+0.21.0-SNAPSHOT (2021-05-20)
+----------------------------
+
+**Added**
+
+**Fixed**
+
+**Dependencies**
+
+**Deprecated**
+
+
```

Comment:
Nit: I think we don't continue the changelog anymore since we automatically log the changes on github via github actions
```suggestion
```